src/lib/npm-client.ts

```diff
@@ -155,20 +154,27 @@ export class NpmPublishClient {
             const body = createTgz(publishedDirectory, reject);
             const metadata = { readme, ...packageJson };
             if (dry) {
-                log("(dry) Skip publish of " + publishedDirectory);
+                log(`(dry) Skip publish of ${publishedDirectory} to ${this.registry}`);
             }
             resolve(dry ? undefined : promisifyVoid(cb => {
-                this.client.publish(npmRegistry, { access: "public", auth: this.auth, metadata, body }, cb);
+                this.client.publish(this.registry, { access: "public", auth: this.auth, metadata, body }, cb);
             }));
         });
     }
 
-    tag(packageName: string, version: string, tag: string): Promise<void> {
-        return promisifyVoid(cb => { this.client.tag(packageUrl(packageName), { version, tag, auth: this.auth }, cb); });
+    tag(packageName: string, version: string, distTag: string, dry: boolean, log: Logger): Promise<void> {
+        return promisifyVoid(cb => {
+            if (dry) {
+                log(`(dry) Skip tag of ${packageName}@${distTag} as ${version}`);
+                cb(undefined);
+            } else {
+                this.client.distTags.add(this.registry, { package: packageName, version, distTag, auth: this.auth }, cb);
```

Comment:
I switched from `tag` to `distTags.add` because it allows me to separately specify the registry URL.